packages/web-components/src/components/ic-radio-group/ic-radio-group.tsx

```diff
@@ -181,21 +184,32 @@ export class RadioGroup {
   private checkOrientation() {
     if (this.initialOrientation === "horizontal") {
       let totalWidth = 0;
-      this.radioOptions.forEach(({ clientWidth }, i, arr) => {
-        totalWidth += clientWidth;
-        if (i < arr.length - 1) totalWidth += 40;
-      });
+      if (Array.isArray(this.radioOptions)) {
+        this.radioOptions.forEach(({ clientWidth }, i, arr) => {
+          totalWidth += clientWidth;
+          if (i < arr.length - 1) totalWidth += 40;
+        });
+      } else {
+        console.warn("radioOptions is undefined or not an array");
```

Comment:
I'm not sure if we need this. `radioOptions` is an internal variable so it's not something we would expose to the consumer. If it fails gracefully, that should be good enough.